### PR TITLE
Opensearchbenchmark introduction

### LINKED ISSUE #479: Create announcement for OpenSearch Benchmark
**Describe the blog post your would like to write**
The blog will be an announcement around OpenSearch Benchmark launch. The blog would highlight what it is, why it is needed and encourage opensearch community members to install, run, request features, etc

**What is the title of the blog post?**
Introducing OpenSearch Benchmark

**Who are the authors?**
- achitojha
- stockholmux

**What is the proposed posting date?**
2021-12-07

### PR DESCRIPTION
**Pull request by achitojha**
### Description
- Post to introduce opensearch benchmark.
- Added author 'achitojha'
 
Note : This PR is a follow-up on [PR#480](https://github.com/opensearch-project/project-website/pull/480). I pushed a commit without the DCO signoff and thus had to create a separate PR.

### Issues Resolved
#479 

### Check List
- [X] Commits are signed per the DCO using --signoff

By submitting this pull request, I confirm that my contribution is made under the terms of the BSD-3-Clause License.

### CODE REVIEW

File: _posts/2021-11-18-introducing-opensearch-benchmark.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+layout: post
+title:  "Introducing OpenSearch Benchmark"
+authors:
+  - achitojha
+  - kyledvs
+date: 2021-11-17 01:01:01 -0700
```
Comment (stockholmux) [inline _posts/2021-11-18-introducing-opensearch-benchmark.md:7]: I think we'll need to change this ;)

**Review by stockholmux — APPROVED** [review]
Do you have any more reviewers you need on this blog?

File: _posts/2021-11-18-introducing-opensearch-benchmark.md (hunk 1 above)
Comment (achitojha) [reply _posts/2021-11-18-introducing-opensearch-benchmark.md:7]: I'll have Phill review it as well. Let's discuss offline what your thoughts are and we can change it accordingly.

**Review by stockholmux — DISMISSED** [review]
LGTM

**Review by treddeni-amazon — APPROVED** [review]

**Comment by dblock** [comment]
I think it would be great to develop lightly on future ideas and plans. 

1. Add a link to the public roadmap of the tool. If there isn't one, create it.
2. Highlight how/when/if OpenSearch-Benchmark is going to be integrated with https://github.com/opensearch-project/opensearch-build, specifically starting from https://github.com/opensearch-project/opensearch-build/issues/126.
3. Be more specific about where you want help from the community. Maybe a list of "good first issue" issues in the repo.

**Review by stockholmux — APPROVED** [review]
✅

**Comment by achitojha** [comment]
@dblock Thanks for your pointers. We intend to publish the roadmap early next week - [tracking issue] https://github.com/opensearch-project/opensearch-benchmark/issues/86

**Pull request merged**